command_plugins/github/plugin.py

```diff
@@ -322,6 +330,66 @@ def add_outside_collab_command(self, data, user_data, collab, org, repo, permiss
                      markdown=True, thread=data["ts"])
 
     @hubcommander_command(
+        name="!SetRepoTeamPermissions",
+        usage="!SetRepoTeamPermissions <Team> <OrgWithRepo> <Repo> <Permission>",
```

Comment:
Let's reorder to:
`!SetRepoPermissions <OrgWithRepo> <Repo> <Team> <Permission>`

Reply:
Done